addons/website/models/ir_ui_view.py

```diff
@@ -140,46 +134,40 @@ def render(self, cr, uid, id_or_xml_id, values=None, engine='ir.qweb', context=N
 
             values = qcontext
 
-        return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
-
-    def _prepare_qcontext(self, cr, uid, context=None):
-        if not context:
-            context = {}
+        return super(View, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
 
-        company = self.pool['res.company'].browse(cr, SUPERUSER_ID, request.website.company_id.id, context=context)
+    @api.model
+    def _prepare_qcontext(self):
+        company = self.env['res.company'].sudo().browse(request.website.company_id.id)
```

Comment:
@rim-odoo ,
As old code it's browsed again with SUPERUSER_ID link : https://github.com/odoo/odoo/pull/8230/files#diff-bf8d854179fb58216be065c91d78d2b6L149 but it's my bad to browse again as before. i am removing it.